app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1078,7 +1078,7 @@ public void onFailure(Call<SusiResponse> call, Throwable t) {
 
     private void updateDatabase(final long prevId, final String message, final boolean isDate, final String date, final String timeStamp , final boolean mine,
```

Comment:
No, I get that. How else would javadoc be created if we don't write javadoc comments. That will be done. I am asking for hosting javadoc on github pages?